karma.browserstack.conf.js

```diff
@@ -5,37 +5,51 @@ const merge = require('deepmerge');
 const defaultPattern = '+(components|helpers|mixins|templates)/**/*.test.js';
```

Comment:
From [the open-wc docs](https://open-wc.org/testing/testing-karma-bs.html) there's some helpers for using BrowserStack... but actually [looking at the code](https://github.com/open-wc/open-wc/blob/master/packages/testing-karma-bs/bs-settings.js) it doesn't really provide anything special, so nevermind!